doc/ref_model/chapters/chapter04.md

```diff
@@ -540,6 +540,47 @@ Whereas:
 <p align="center"><img src="../figures/ch04_one_stop_shop.PNG" alt="one_stop_shop" title="One Stop Shop" width="100%"/></p>
 <p align="center"><b>Figure 4-3:</b> Infrastructure Profiles Catalogue</p>
 
+
+<a name="4.2.7.2"></a>
+#### 4.2.7.2 Backwards Compatibility
+The Reference Model (RM) specification describes a complete infrastructure abstraction including a set of cloud infrastructure hardware and software profiles and compute flavours offered to workloads. The set of defined profiles and flavours will evolve along the releases but at the same time the existing workloads need to be supported.  This means that any CNTT deployed cloud should be backwards compatible and support profiles and flavours from the latest three CNTT releases (N-2, N-1, N) as presented in Figure 4-4. 
+
+
+<p align="center"><img src="../figures/ch04-Backwards-compatibility_overview.png" alt="backwards compatibility" title="backwards compatibility" width="100%"/></p>
+<p align="center"><b>Figure 4-4:</b> Backwards Compatibility</p>
+
+Cloud Infrastructure profiles that are available in CNTT release N deployment can be divided into two categories:
+
+ 1. Cloud infrastructure profiles that are part of CNTT release N. These can be either
+    * new profiles defined in release N or
+    * profiles that were defined in earlier releases but are also supported in release N
+ 2. Cloud infrastructure profiles from releases N-1 and N-2 that are deployed only because of backwards compatibility, these profiles are not part of CNTT release N definition.
+
+Notice1: it is possible that in CNTT release N deployment there is release N-1 profile with exception because of backwards compatibility and in release N the same profile but without the exception. 
```

Comment:
Notice 1 -- IMO these are then 2 different profiles; the relase N will consist of the new profile w/o exception as part of its release but will continue to support the version with exceptions for backward compatibility.

Reply:
Soo, you mean that Notice 1 is not valid as if the exceptions of a profile are removed it becomes a new profile?

Reply:
@CsatariGergely I think so. Also, their "proper names" will be different -- the version w/o the exceptions should have a new release number N as part of their name.

Reply:
Okey, Notice1 is removed now.